.doc/installation.asciidoc

```diff
@@ -48,3 +48,32 @@ END
 
 go run main.go
 ------------------------------------
+
+
+[discrete]
+=== {es} Version Compatibility
+
+The client major versions correspond to the {es} major versions: to connect to 
+{es} `7.x`, use a `7.x` version of the client, to connect to {es} `6.x`, use a 
+`6.x` version of the client, and so on.
+
```

Comment:
Can we add wording, that while a `5.x` client is available, it's no longer actively maintained given that neither is any 5.x version of Elasticsearch. Could link to the EOL policy: https://www.elastic.co/support/eol

Reply:
Thanks @philkra! I've added a note via 2bdf729.